_source/_posts/2017-12-04-use-kong-gateway-to-centralize-authentication.md

```diff
@@ -6,8 +6,12 @@ by: advocate
 communities: [java]
 tags: [kong, api-gateway, api, gateway, spring, spring-boot]
 type: conversion
+changelog: 
+- 2021-03-25: Updated with refreshed video and new github repo reference. Changes to this article can be viewed in [oktadeveloper/okta-blog#570](https://github.com/oktadeveloper/okta-blog/pull/570).
 ---
 
+**NOTE:** The video and code in this post has just been updated for 2021!
```

Comment:
I'm not sure if you need this since "last updated" is displayed at the top. Feel free to keep it though.